Mathlib/Data/List/Destutter.lean

```diff
@@ -148,4 +151,116 @@ theorem destutter_eq_nil : ∀ {l : List α}, destutter R l = [] ↔ l = []
   | [] => Iff.rfl
   | _ :: l => ⟨fun h => absurd h <| l.destutter'_ne_nil R, fun h => nomatch h⟩
 
+variable {R}
+
+/-- For a relation-preserving map, `destutter'` commutes with `map`. -/
```

Comment:
```suggestion
/-- For a relation-preserving map, `destutter'` commutes with `map`. -/
```
Doc-strings need updating to remove the prime.

Reply:
I assume you mean the prime in `destutter'`? Your suggestion above doesn't seem to contain any change 😄